inji-web/src/__tests__/components/DataShare/DataShareContent.test.tsx

```diff
@@ -0,0 +1,58 @@
+import {fireEvent, render, screen} from "@testing-library/react";
+import {DataShareContent} from "../../../components/DataShare/DataShareContent";
+import {reduxStore} from "../../../redux/reduxStore";
+import {Provider} from "react-redux";
+
+describe("Test the Layout of the Expiry Content", () => {
+
+    const customMockFn = jest.fn();
+    beforeEach(() => {
+        jest.mock("react-i18next", () => ({
+            useTranslation: () => ({
+                t: (key: string) => key,
+            }),
+        }));
+        render(<Provider store={reduxStore}>
+            <DataShareContent credentialName={"credentialName"} credentialLogo={"credentialLogo"} setCustom={customMockFn} />
+        </Provider>);
+    } )
+
+    test("Test the presence of the Outer Container", ()=>{
+        const document = screen.getByTestId("DataShareContent-Outer-Container");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Outer Title", ()=>{
+        const document = screen.getByTestId("DataShareContent-Outer-Title");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Issuer Logo", ()=>{
+        const document = screen.getByTestId("DataShareContent-Issuer-Logo");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Issuer Name", ()=>{
+        const document = screen.getByTestId("DataShareContent-Issuer-Name");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Consent Container", ()=>{
+        const document = screen.getByTestId("DataShareContent-Consent-Container");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the Validity Times Dropdown should not show custom as selected option at first", ()=>{
+        const selectedDocument = screen.getByTestId("DataShareContent-Selected-Validity-Times");
+        expect(selectedDocument).not.toHaveTextContent("Custom");
+        expect(selectedDocument).toHaveTextContent("Once");
+        const document = screen.queryByTestId("DataShareContent-Validity-Times-DropDown");
+        expect(document).not.toBeInTheDocument();
```

Comment:
Cover as single snapshot test as there is no action performed and input is same for all tests

Reply:
Snapshot Test we will take it up as part of different card mentioned.